packages/docs-theme/src/styles/_examples.scss

```diff
@@ -85,6 +86,10 @@ $dark-options-background-color: $dark-gray5;
     display: none;
   }
 
+  .#{$ns}-heading:not(:first-child) {
```

Comment:
oh whoops, that was meant to be in the other PR, but I guess it's fine to leave here as it improves spacing in options